src/test/java/com/radixdlt/crypto/SchnorrSignature.java

```diff
@@ -0,0 +1,8 @@
+package com.radixdlt.crypto;
```

Comment:
Missing header comment.

Reply:
Actually, I think this should be removed.  It's not part of the work required now.

Reply:
@clstrfsck it is in test folder, used for tests.

Reply:
Oh, yup, so it is 🤦‍♂

Just the header comment then.